src/Traits/ExceptionsTestTrait.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace RonasIT\Support\Traits;
+
+trait ExceptionsTestTrait
+{
+    protected function assertExceptionThrew(string $className, string $message): void
```

Comment:
let's add the strict mode to the logic and change regexp accordingly to it

```suggestion
    protected function assertExceptionThrew(string $className, string $message, bool isStrinct = true): void
```